### PR TITLE
Update main for Codacy checks

### PR DESCRIPTION
**Pull request by ritchiexia**

### CODE REVIEW

File: backend/tests/userTests.test.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+test('Scan DB for user email', () => {
+    const req = 'test@test.com';
+    const res = {"userID": 1};
+
+    axios.get = jest.fn().mockResolvedValue(res);
+    return axios.get(UserDBURL + '/scanDB?email=' + req).then(response => expect(response).toEqual(res))
+});
+
+test('Store user info', () => {
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:55]: Codacy has a fix for the issue: [Replace `'Store·user·info'` with `"Store·user·info"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
test("Store user info", () => {
```

File: backend/tests/restrictionTests.test.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+test('Testing add restriction', () => {
+	const req = {'userID': 1, 'dietaryRestrictions': ['apple', 'orange']};
+	const res = {'result': 'Successfully added restriction'};
+
+    axios.put = jest.fn().mockResolvedValue(res);
+    return axios.put(UserStoreURL + '/addRestrictions', {
+        method: 'PUT',
+        headers: {
+            'Content-Type': 'application/json'
+        },
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:18]: Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
      },
```

File: backend/tests/userTests.test.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    const req = {'email': 'test4@test.com'};
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:56]: Codacy has a fix for the issue: [Replace `····const·req·=·{'email':·'test4@test.com'` with `··const·req·=·{·email:·"test4@test.com"·`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  const req = { email: "test4@test.com" };
```

File: backend/tests/restrictionTests.test.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+	axios.get = jest.fn().mockResolvedValue(res);
+	return axios.get(UserStoreURL + '/getRestrictions?userid=' + req).then(response => expect(response).toEqual(res));
+});
+
+test('Testing add restrictions for DB', () => {
+    const req = {'userID': 1, 'ingredient': 'apple'};
+	const res = {'result': 'Successfully added restriction'};
+
+    axios.put = jest.fn().mockResolvedValue(res);
+    return axios.put(UserDBURL + '/addToDietaryRestrictions', {
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:51]: Codacy has a fix for the issue: [Replace `··return·axios.put(UserDBURL·+·'/addToDietaryRestrictions'` with `return·axios⏎····.put(UserDBURL·+·"/addToDietaryRestrictions"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  return axios
```

File: backend/tests/restrictionTests.test.js [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        method: 'PUT',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Testing get restriction', () => {
+	const req = '1';
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:39]: Codacy has a fix for the issue: [Replace `↹const·req·=·'1'` with `··const·req·=·"1"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  const req = "1";
```

File: backend/tests/userTests.test.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Get tokens', () => {
+    const req = '1, 2, 3';
+    const res = [{'userID': 1, 'token': 'a', "_id": ObjectID("17C6E3C032F89045AD746684")}, {'userID': 2, 'token': 'b', "_id": ObjectID("17C6E3C032F89045AD746685")}, {'userID': 3, 'token': 'c', "_id": ObjectID("17C6E3C032F89045AD746686")}];
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:85]: Codacy has a fix for the issue: [Replace `····const·res·=·[{'userID':·1,·'token':·'a',·"_id":·ObjectID("17C6E3C032F89045AD746684")},·{'userID':·2,·'token':·'b',·"_id":·ObjectID("17C6E3C032F89045AD746685")},·{'userID':·3,·'token':·'c',·"_id":·ObjectID("17C6E3C032F89045AD746686")}` with `··const·res·=·[⏎····{·userID:·1,·token:·"a",·_id:·ObjectID("17C6E3C032F89045AD746684")·},⏎····{·userID:·2,·token:·"b",·_id:·ObjectID("17C6E3C032F89045AD746685")·},⏎····{·userID:·3,·token:·"c",·_id:·ObjectID("17C6E3C032F89045AD746686")·},⏎··`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  const res = [
```

File: backend/tests/restrictionTests.test.js [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Testing delete restrictions for DB', () => {
+    const req = {'userID': 1, 'dietaryRestrictions': ['apple', 'banana']};
+	const res = {'result': 'Successfully added restriction'};
+
+    axios.put = jest.fn().mockResolvedValue(res);
+    return axios.put(UserDBURL + '/deleteFromDietaryRestrictions', {
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:65]: Codacy has a fix for the issue: [Replace `··return·axios.put(UserDBURL·+·'/deleteFromDietaryRestrictions'` with `return·axios⏎····.put(UserDBURL·+·"/deleteFromDietaryRestrictions"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  return axios
```

File: backend/tests/userTests.test.js [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+test('Get user tokens', () => {
+    const req = '1, 2, 3';
+    const res = [{'userID': 1, 'token': 'a'}, {'userID': 2, 'token': 'b'}, {'userID': 3, 'token': 'c'}];
+
+    axios.get = jest.fn().mockResolvedValue(res);
+    return axios.get(UserStoreURL + '/getUserTokens?userids=' + req).then(response => expect(response).toEqual(res))
+});
+
+
+test('Store user token', () => {
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:23]: Codacy has a fix for the issue: [Replace `'Store·user·token'` with `"Store·user·token"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
test("Store user token", () => {
```

File: backend/tests/userTests.test.js [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Get tokens', () => {
+    const req = '1, 2, 3';
+    const res = [{'userID': 1, 'token': 'a', "_id": ObjectID("17C6E3C032F89045AD746684")}, {'userID': 2, 'token': 'b', "_id": ObjectID("17C6E3C032F89045AD746685")}, {'userID': 3, 'token': 'c', "_id": ObjectID("17C6E3C032F89045AD746686")}];
+
+    axios.get = jest.fn().mockResolvedValue(res);
+    return axios.get(UserDBURL + '/getTokens?userids=' + req).then(response => expect(response).toEqual(res))
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:88]: Codacy has a fix for the issue: [Replace `··return·axios.get(UserDBURL·+·'/getTokens?userids='·+·req).then(response·=>·expect(response).toEqual(res))` with `return·axios⏎····.get(UserDBURL·+·"/getTokens?userids="·+·req)⏎····.then((response)·=>·expect(response).toEqual(res));`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  return axios
```

File: backend/tests/restrictionTests.test.js [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+
+test('Testing add restriction', () => {
+	const req = {'userID': 1, 'dietaryRestrictions': ['apple', 'orange']};
+	const res = {'result': 'Successfully added restriction'};
+
+    axios.put = jest.fn().mockResolvedValue(res);
+    return axios.put(UserStoreURL + '/addRestrictions', {
+        method: 'PUT',
+        headers: {
+            'Content-Type': 'application/json'
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:17]: Codacy has a fix for the issue: [Replace `····'Content-Type':·'application/json'` with `"Content-Type":·"application/json",`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
        "Content-Type": "application/json",
```

File: backend/tests/userTests.test.js [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Check user exists', () => {
+    const req = 'test@test.com';
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:38]: Codacy has a fix for the issue: [Replace `····const·req·=·'test@test.com'` with `··const·req·=·"test@test.com"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  const req = "test@test.com";
```

File: backend/tests/userTests.test.js [hunk 12]
```diff
@@ -0,0 +1,89 @@
+// M6 Q3
+//import {jest} from '@jest/globals';
+//import axios from 'axios';
+var axios = require('axios');
+const { ObjectID } = require('bson');
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:5]: Codacy has a fix for the issue: [Replace `'bson'` with `"bson"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
const { ObjectID } = require("bson");
```

File: backend/tests/userTests.test.js [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+    axios.get = jest.fn().mockResolvedValue(res);
+    return axios.get(UserStoreURL + '/getUserTokens?userids=' + req).then(response => expect(response).toEqual(res))
+});
+
+
+test('Store user token', () => {
+    const req = {'userID': 1, 'token': 'a'};
+    const res = {'result': 'Sucessfully stored user token'};
+
+    axios.post = jest.fn().mockResolvedValue(res);
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:27]: Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  axios.post = jest.fn().mockResolvedValue(res);
```

File: backend/tests/userTests.test.js [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Store token', () => {
+    const req = {'userID': 4, 'token': 'd'};
+    const res = {'result': "New user's token has been added to DB"};
+
+    axios.post = jest.fn().mockResolvedValue(res);
+    return axios.post(UserDBURL + '/storeUserToken', {
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:74]: Codacy has a fix for the issue: [Replace `··return·axios.post(UserDBURL·+·'/storeUserToken'` with `return·axios⏎····.post(UserDBURL·+·"/storeUserToken"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  return axios
```

File: backend/tests/restrictionTests.test.js [hunk 15]
```diff
@@ -0,0 +1,80 @@
...
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Testing get restriction for DB', () => {
+	const req = '1';
+	const res = {data: ['banana', 'orange']};
+
+	axios.get = jest.fn().mockResolvedValue(res);
+	return axios.get(UserDBURL + '/getDietaryRestrictions?userid=' + req).then(response => expect(response).toEqual(res));
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:79]: Codacy has a fix for the issue: [Replace `↹return·axios.get(UserDBURL·+·'/getDietaryRestrictions?userid='·+·req).then(response` with `··return·axios⏎····.get(UserDBURL·+·"/getDietaryRestrictions?userid="·+·req)⏎····.then((response)`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  return axios
```

File: backend/tests/userTests.test.js [hunk 16]
```diff
@@ -0,0 +1,89 @@
...
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Store token', () => {
+    const req = {'userID': 4, 'token': 'd'};
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:70]: Codacy has a fix for the issue: [Replace `····const·req·=·{'userID':·4,·'token':·'d'` with `··const·req·=·{·userID:·4,·token:·"d"·`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  const req = { userID: 4, token: "d" };
```

File: backend/tests/userTests.test.js [hunk 17]
```diff
@@ -0,0 +1,89 @@
...
+    const req = '1, 2, 3';
+    const res = [{'userID': 1, 'token': 'a'}, {'userID': 2, 'token': 'b'}, {'userID': 3, 'token': 'c'}];
+
+    axios.get = jest.fn().mockResolvedValue(res);
+    return axios.get(UserStoreURL + '/getUserTokens?userids=' + req).then(response => expect(response).toEqual(res))
+});
+
+
+test('Store user token', () => {
+    const req = {'userID': 1, 'token': 'a'};
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:24]: Codacy has a fix for the issue: [Replace `····const·req·=·{'userID':·1,·'token':·'a'` with `··const·req·=·{·userID:·1,·token:·"a"·`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  const req = { userID: 1, token: "a" };
```

File: backend/tests/userTests.test.js [hunk 18]
```diff
@@ -0,0 +1,89 @@
+// M6 Q3
+//import {jest} from '@jest/globals';
+//import axios from 'axios';
+var axios = require('axios');
+const { ObjectID } = require('bson');
+
+const UserStoreURL = 'http://20.53.224.7:8082';
+const UserDBURL = 'http://20.53.224.7:8081';
+
+jest.mock('axios');
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:10]: Codacy has a fix for the issue: [Replace `'axios'` with `"axios"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
jest.mock("axios");
```

File: backend/tests/restrictionTests.test.js [hunk 19]
```diff
@@ -0,0 +1,80 @@
...
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Testing get restriction', () => {
+	const req = '1';
+	const res = {data: ['banana', 'orange']};
+
+	axios.get = jest.fn().mockResolvedValue(res);
+	return axios.get(UserStoreURL + '/getRestrictions?userid=' + req).then(response => expect(response).toEqual(res));
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:43]: Codacy has a fix for the issue: [Replace `↹return·axios.get(UserStoreURL·+·'/getRestrictions?userid='·+·req).then(response` with `··return·axios⏎····.get(UserStoreURL·+·"/getRestrictions?userid="·+·req)⏎····.then((response)`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  return axios
```

File: backend/tests/userTests.test.js [hunk 20]
```diff
@@ -0,0 +1,89 @@
+// M6 Q3
+//import {jest} from '@jest/globals';
+//import axios from 'axios';
+var axios = require('axios');
+const { ObjectID } = require('bson');
+
+const UserStoreURL = 'http://20.53.224.7:8082';
+const UserDBURL = 'http://20.53.224.7:8081';
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:8]: Codacy has a fix for the issue: [Replace `'http://20.53.224.7:8081'` with `"http://20.53.224.7:8081"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
const UserDBURL = "http://20.53.224.7:8081";
```

File: backend/tests/restrictionTests.test.js (hunk 4 above)
Comment (ritchiexia) [inline backend/tests/restrictionTests.test.js:51]: This looks like the suggestion would return incorrectly. It is removing the put() axios call which is required.

File: backend/tests/restrictionTests.test.js [hunk 21]
```diff
@@ -0,0 +1,80 @@
+//import axios from 'axios';
+var axios = require('axios')
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:2]: Codacy has a fix for the issue: [Replace `'axios')` with `"axios");`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
var axios = require("axios");
```

File: backend/tests/userTests.test.js [hunk 22]
```diff
@@ -0,0 +1,89 @@
...
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Get tokens', () => {
+    const req = '1, 2, 3';
+    const res = [{'userID': 1, 'token': 'a', "_id": ObjectID("17C6E3C032F89045AD746684")}, {'userID': 2, 'token': 'b', "_id": ObjectID("17C6E3C032F89045AD746685")}, {'userID': 3, 'token': 'c', "_id": ObjectID("17C6E3C032F89045AD746686")}];
+
+    axios.get = jest.fn().mockResolvedValue(res);
+    return axios.get(UserDBURL + '/getTokens?userids=' + req).then(response => expect(response).toEqual(res))
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:88]: Codacy has a fix for the issue: [Replace `··return·axios.get(UserDBURL·+·'/getTokens?userids='·+·req).then(response·=>·expect(response).toEqual(res))` with `return·axios⏎····.get(UserDBURL·+·"/getTokens?userids="·+·req)⏎····.then((response)·=>·expect(response).toEqual(res));`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  return axios
```

File: backend/tests/userTests.test.js [hunk 23]
```diff
@@ -0,0 +1,89 @@
...
+});
+
+
+test('Store user token', () => {
+    const req = {'userID': 1, 'token': 'a'};
+    const res = {'result': 'Sucessfully stored user token'};
+
+    axios.post = jest.fn().mockResolvedValue(res);
+    return axios.post(UserStoreURL + '/storeUserToken', {
+        method: 'POST',
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:29]: Codacy has a fix for the issue: [Replace `··method:·'POST'` with `method:·"POST"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
      method: "POST",
```

File: backend/tests/userTests.test.js [hunk 24]
```diff
@@ -0,0 +1,89 @@
...
+        headers: {
+            'Content-Type': 'application/json'
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:31]: Codacy has a fix for the issue: [Replace `····'Content-Type':·'application/json'` with `"Content-Type":·"application/json",`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
        "Content-Type": "application/json",
```

File: backend/tests/userTests.test.js (hunk 11 above)
Comment (Harvir14) [inline backend/tests/userTests.test.js:38]: Codacy has a fix for the issue: [Replace `····const·req·=·'test@test.com'` with `··const·req·=·"test@test.com"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  const req = "test@test.com";
```

File: backend/tests/userTests.test.js [hunk 25]
```diff
@@ -0,0 +1,89 @@
...
+    return axios.get(UserStoreURL + '/getUserTokens?userids=' + req).then(response => expect(response).toEqual(res))
+});
+
+
+test('Store user token', () => {
+    const req = {'userID': 1, 'token': 'a'};
+    const res = {'result': 'Sucessfully stored user token'};
+
+    axios.post = jest.fn().mockResolvedValue(res);
+    return axios.post(UserStoreURL + '/storeUserToken', {
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:28]: Codacy has a fix for the issue: [Replace `··return·axios.post(UserStoreURL·+·'/storeUserToken'` with `return·axios⏎····.post(UserStoreURL·+·"/storeUserToken"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  return axios
```

File: backend/tests/restrictionTests.test.js [hunk 26]
```diff
@@ -0,0 +1,80 @@
...
+jest.mock('axios');
+
+const UserStoreURL = 'http://20.53.224.7:8082';
+const UserDBURL = 'http://20.53.224.7:8081';
+
+test('Testing add restriction', () => {
+	const req = {'userID': 1, 'dietaryRestrictions': ['apple', 'orange']};
+	const res = {'result': 'Successfully added restriction'};
+
+    axios.put = jest.fn().mockResolvedValue(res);
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:13]: Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  axios.put = jest.fn().mockResolvedValue(res);
```

File: backend/tests/restrictionTests.test.js [hunk 27]
```diff
@@ -0,0 +1,80 @@
...
+        method: 'PUT',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Testing get restriction for DB', () => {
+	const req = '1';
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:75]: Codacy has a fix for the issue: [Replace `↹const·req·=·'1'` with `··const·req·=·"1"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  const req = "1";
```

File: backend/tests/userTests.test.js [hunk 28]
```diff
@@ -0,0 +1,89 @@
...
+    return axios.post(UserDBURL + '/storeUserInfo', {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Store token', () => {
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:69]: Codacy has a fix for the issue: [Replace `'Store·token'` with `"Store·token"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
test("Store token", () => {
```

File: backend/tests/restrictionTests.test.js [hunk 29]
```diff
@@ -0,0 +1,80 @@
...
+    return axios.put(UserStoreURL + '/deleteRestrictions', {
+        method: 'PUT',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Testing get restriction', () => {
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:38]: Codacy has a fix for the issue: [Replace `'Testing·get·restriction'` with `"Testing·get·restriction"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
test("Testing get restriction", () => {
```

File: backend/tests/userTests.test.js [hunk 30]
```diff
@@ -0,0 +1,89 @@
...
+    const res = [{'userID': 1, 'token': 'a'}, {'userID': 2, 'token': 'b'}, {'userID': 3, 'token': 'c'}];
+
+    axios.get = jest.fn().mockResolvedValue(res);
+    return axios.get(UserStoreURL + '/getUserTokens?userids=' + req).then(response => expect(response).toEqual(res))
+});
+
+
+test('Store user token', () => {
+    const req = {'userID': 1, 'token': 'a'};
+    const res = {'result': 'Sucessfully stored user token'};
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:25]: Codacy has a fix for the issue: [Replace `··const·res·=·{'result':·'Sucessfully·stored·user·token'` with `const·res·=·{·result:·"Sucessfully·stored·user·token"·`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  const res = { result: "Sucessfully stored user token" };
```

File: backend/tests/restrictionTests.test.js [hunk 31]
```diff
@@ -0,0 +1,80 @@
...
+const UserDBURL = 'http://20.53.224.7:8081';
+
+test('Testing add restriction', () => {
+	const req = {'userID': 1, 'dietaryRestrictions': ['apple', 'orange']};
+	const res = {'result': 'Successfully added restriction'};
+
+    axios.put = jest.fn().mockResolvedValue(res);
+    return axios.put(UserStoreURL + '/addRestrictions', {
+        method: 'PUT',
+        headers: {
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:16]: Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
      headers: {
```

File: backend/tests/restrictionTests.test.js [hunk 32]
```diff
@@ -0,0 +1,80 @@
+//import axios from 'axios';
+var axios = require('axios')
+
+jest.mock('axios');
+
+const UserStoreURL = 'http://20.53.224.7:8082';
+const UserDBURL = 'http://20.53.224.7:8081';
+
+test('Testing add restriction', () => {
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:9]: Codacy has a fix for the issue: [Replace `'Testing·add·restriction'` with `"Testing·add·restriction"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
test("Testing add restriction", () => {
```

File: backend/tests/restrictionTests.test.js [hunk 33]
```diff
@@ -0,0 +1,80 @@
+//import axios from 'axios';
+var axios = require('axios')
+
+jest.mock('axios');
+
+const UserStoreURL = 'http://20.53.224.7:8082';
+const UserDBURL = 'http://20.53.224.7:8081';
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:7]: Codacy has a fix for the issue: [Replace `'http://20.53.224.7:8081'` with `"http://20.53.224.7:8081"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
const UserDBURL = "http://20.53.224.7:8081";
```

File: backend/tests/restrictionTests.test.js [hunk 34]
```diff
@@ -0,0 +1,80 @@
...
+    return axios.put(UserDBURL + '/addToDietaryRestrictions', {
+        method: 'PUT',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Testing delete restrictions for DB', () => {
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:60]: Codacy has a fix for the issue: [Replace `'Testing·delete·restrictions·for·DB'` with `"Testing·delete·restrictions·for·DB"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
test("Testing delete restrictions for DB", () => {
```

File: backend/tests/userTests.test.js [hunk 35]
```diff
@@ -0,0 +1,89 @@
...
+    const req = 'test@test.com';
+    const res = {'userID': 1};
+
+    axios.get = jest.fn().mockResolvedValue(res);
+    return axios.get(UserStoreURL + '/checkUserExists?email=' + req).then(response => expect(response).toEqual(res))
+});
+
+//UserDB.js tests
+
+test('Scan DB for user email', () => {
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:47]: Codacy has a fix for the issue: [Replace `'Scan·DB·for·user·email'` with `"Scan·DB·for·user·email"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
test("Scan DB for user email", () => {
```

File: backend/tests/restrictionTests.test.js [hunk 36]
```diff
@@ -0,0 +1,80 @@
...
+    return axios.put(UserStoreURL + '/addRestrictions', {
+        method: 'PUT',
+        headers: {
+            'Content-Type': 'application/json'
+      },
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Testing delete restriction', () => {
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:23]: Codacy has a fix for the issue: [Replace `'Testing·delete·restriction'` with `"Testing·delete·restriction"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
test("Testing delete restriction", () => {
```

File: backend/tests/userTests.test.js [hunk 37]
```diff
@@ -0,0 +1,89 @@
...
+    return axios.post(UserStoreURL + '/storeUserToken', {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Check user exists', () => {
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:37]: Codacy has a fix for the issue: [Replace `'Check·user·exists'` with `"Check·user·exists"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
test("Check user exists", () => {
```

File: backend/tests/restrictionTests.test.js [hunk 38]
```diff
@@ -0,0 +1,80 @@
...
+        },
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Testing get restriction', () => {
+  const req = "1";
+	const res = {data: ['banana', 'orange']};
+
+	axios.get = jest.fn().mockResolvedValue(res);
```
Comment (Harvir14) [inline backend/tests/restrictionTests.test.js:42]: Codacy has a fix for the issue: [Replace `↹` with `··`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
  axios.get = jest.fn().mockResolvedValue(res);
```

File: backend/tests/userTests.test.js [hunk 39]
```diff
@@ -0,0 +1,89 @@
...
+    return axios.post(UserDBURL + '/storeUserToken', {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Get tokens', () => {
```
Comment (Harvir14) [inline backend/tests/userTests.test.js:83]: Codacy has a fix for the issue: [Replace `'Get·tokens'` with `"Get·tokens"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
test("Get tokens", () => {
```

**Pull request merged**